session_0/fizz_buzz.rb

```diff
@@ -1,41 +1,22 @@
-# Fizz Buzz is a word game for children to teach them about division.
-#
-# We will use it to learn about transforming traditional C-like for
-# loops into ruby code and a bit of error handling.
-#
-# The rules of the game are as follows:
-# - The first player says the number "1".
-# - Each subsequent player counts upwards in turn.
-# - Numbers divisible by 3 are replaced by the word "Fizz"
-# - Numbers divisible by 5 are replaced by the word "Buzz"
-# - Numbers divisible by both 3 and 5 are replaced by the word "FizzBuzz"
-#
-# For example, a round of Fizz Buzz would go as follows:
-#
-# 1, 2, Fizz, 4, Buzz, Fizz, 7, 8, Fizz, Buzz, 11, Fizz, 13, 14,
-# FizzBuzz, 16, 17, Fizz, 19, Buzz, Fizz, 22, 23, Fizz, Buzz, 26, Fizz,
-# 28, 29, FizzBuzz, 31, 32, Fizz, 34, Buzz, Fizz, ...
-#
-# We will take a look at generalization of Fizz Buzz - instead of
-# numbers divisible by 3 or 5, we will let the user decide.
-# They are denoted by keyword arguments (see README.md for reference)
-# `x` and `y`.
-#
-# The first keyword argument `n` represents the number of turns.
-#
-# We will be returning an array of strings, containing what each player
-# says on their turn.
-#
-# For example, fizz_buzz(n: 6, x: 2, y: 3) should return the following:
+# Fizz Buzz
```

Comment:
Please don't push session 0 an 1 again - using branches and pushing code correctly is part of learning how to use Git.

Can you remove them from the MR?

Reply:
I am sorry. I will remove them.